gtsam_unstable/gtsam_unstable.i

```diff
@@ -797,6 +805,17 @@ virtual class ProjectionFactorPPPC : gtsam::NoiseModelFactor {
 typedef gtsam::ProjectionFactorPPPC<gtsam::Pose3, gtsam::Point3, gtsam::Cal3_S2> ProjectionFactorPPPCCal3_S2;
 typedef gtsam::ProjectionFactorPPPC<gtsam::Pose3, gtsam::Point3, gtsam::Cal3DS2> ProjectionFactorPPPCCal3DS2;
 
+#include <gtsam_unstable/nonlinear/LinearizedFactor.h>
+
+virtual class LinearizedGaussianFactor : gtsam::NonlinearFactor {
+  // LinearizedGaussianFactor();
+  // LinearizedGaussianFactor(const boost::shared_ptr<gtsam::GaussianFactor>& gaussian, const gtsam::Values& lin_points);
+};
+
+virtual class LinearizedHessianFactor : gtsam::LinearizedGaussianFactor {
+  LinearizedHessianFactor();
+  LinearizedHessianFactor(const boost::shared_ptr<gtsam::HessianFactor>& hessian, const gtsam::Values& lin_points);
```

Comment:
class not closed here leading to build failure

Reply:
ouch, thanks alex! Fixed.